### DISCUSSION TITLE
Hỏi về cách convert ngược mã MD5

### DISCUSSION THREAD

**Discussion by normamorales** [codingreshapefuture/Community#22] [Q&A] [answered]
Hỏi về cách convert ngược mã MD5

**Comment by lucthienphong1120** [comment, marked as answer]
hash MD5 là encrypt 1 chiều, hash nghĩa là “băm”, thịt đã “băm” nát rồi thì không khôi phục lại như cũ được, data hash xong rồi thì không decrypt được.
dùng rainbow table để dò mã hash, rainbow table là môt database chứa một số mã hash đã biết trước key. Nếu key là một chuỗi phổ biến VD: 123, Iloveyou, admin,… thì có thể “decrypt” bằng rainbow table.

**Reply by normamorales** [reply]
good solution, thanks
